### PR TITLE
data: EndGaurav

### PR DESCRIPTION
**Pull request by EndGaurav**
## Check List (Check all the applicable boxes) 

- [ ] My code follows the code style of this project.
- [ ] My change requires changes to the documentation.
- [ ] I have updated the documentation accordingly.
- [ ] All new and existing tests passed.
- [ ] This PR does not contain plagiarized content.
- [ ] The title of my pull request is a short description of the requested changes.

## Note to reviewers

<a href="https://gitpod.io/#https://github.com/EddieHubCommunity/LinkFree/pull/4630"></a>

### CODE REVIEW

**Comment by ChinmayMhatre** [comment]
Hey @EndGaurav , move the json file to the data directory

**Comment by EndGaurav** [comment]
Hey @ChinmayMhatre please can you tell me how can i move that file to the data directory ?

**Comment by ChinmayMhatre** [comment]
Have you setup Linkfree locally? Or did you do it on github?

**Review by priyankarpal — CHANGES_REQUESTED** [review]
hey, @EndGaurav you mistakenly created `json` file in the main folder. You have to change it to `data/EndGaurav.json` To change this check the [official doc ](https://linkfree.eddiehub.io/docs/quickstart). If you're facing any doubts you can ask. ✌️

**Review by Hrishi5111998 — CHANGES_REQUESTED** [review]
hey @EndGaurav there is typo in your bio newbier should be **newbie**

**Comment by EndGaurav** [comment]
Hey @ChinmayMhatre i have done it on github. locally i can do that locally by forking. so please tell me will i have to create a new pull request for putting into data directory ?

**Comment by EndGaurav** [comment]
@Hrishi5111998 there is no typo in my bio bro. where have you seen that ?

**Review by priyankarpal — APPROVED** [review]
Looks Good 🎉

**Comment by EndGaurav** [comment]
Hey @kumarsonsoff3 i have moved that file in data directory.

File: data/EndGaurav.json [hunk 1]
```diff
@@ -0,0 +1,18 @@
+{
+  "name": "Gaurav Singh Jethuri",
+  "displayStatsPublic": true,
```
Comment (kumarsonsoff3) [inline data/EndGaurav.json:3]: This `displayStatsPublic` key is no longer needed as of our latest update, so please update it accordingly. 
```suggestion
```

**Review by kumarsonsoff3 — CHANGES_REQUESTED** [review]

**Review by EndGaurav — COMMENTED** [review]
have you deleted that "displayStatsPublic" : true ? or will i have to do thar. sorry i am little bit unable to understand that what exactly happening. can you tell me ?

**Comment by kumarsonsoff3** [comment]
> have you deleted that "displayStatsPublic" : true ? or will i have to do thar. sorry i am little bit unable to understand that what exactly happening. can you tell me ?

No worries, it does happen, I asked you to delete that field because we no longer need that. Hence you did the same. Now it's good to go! 🚀

**Review by kumarsonsoff3 — APPROVED** [review]

**Comment by EndGaurav** [comment]
very  very thanks all of you guyss for helping me in this. can you help me how from here how can i get more involve in open source?

**Comment by kumarsonsoff3** [comment]
> very very thanks all of you guyss for helping me in this. can you help me how from here how can i get more involve in open source?

Hey Gaurav, you can check out other good first issues you can work on, in LinkFree. Also, there are tonnes of projects out there you can contribute to. You can use https://github.com/search/advanced to find projects that fits you more effectively. 

Any questions, we can chat on EddieHub Discord: http://discord.eddiehub.org/

**Comment by EndGaurav** [comment]
okay, thanks @kumarsonsoff3 .

**Pull request merged**